mozci/console/commands/push.py

```diff
@@ -124,7 +126,9 @@ def handle(self):
         for push in pushes:
             try:
                 classification, regressions = push.classify(
-                    confidence_medium=medium_conf, confidence_high=high_conf
+                    confidence_medium=medium_conf,
+                    confidence_high=high_conf,
```

Comment:
The names of the arguments changed, can you rebase?

Reply:
okay done